Sushma branch

sushma-ananda opened the pull request.

timobaumann commented:
Hi Sushma. You have added class files with this merge (lots of classes in a build directory). 
Please remove them.

sushma-ananda commented:
Hi Timo, thanks, This is done now. updated gitignore : commit# d8a664d470a8633c0aa6a45ced135d10d692c865
removed class files and few other changes: 9666980c517987512315fc643f5ff31e7f3b9208 .

timobaumann commented:
Hi Sushma, there's a funny directory "C:\\Users\\fpecune\\Desktop\\Logs" 
checked in since your commit 550a83. Can you please remove that (and the 
files therein?)
@Florian, this file was probably created by code in 
./src/main/java/edu/cmu/inmind/multiuser/sara/component/UserModelComponent.java 
. Can you stop that from doing this?
Thanks, t.

Am 13.06.2017 um 15:42 schrieb sushma-ananda:
>
> Hi Timo, thanks, This is done now. updated gitignore : commit# d8a664d 
> <https://github.com/fpecune/MUF-inMind-April2017-master/commit/d8a664d470a8633c0aa6a45ced135d10d692c865>
> removed class files and few other changes: 9666980 
> <https://github.com/fpecune/MUF-inMind-April2017-master/commit/9666980c517987512315fc643f5ff31e7f3b9208> 
> .
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/fpecune/MUF-inMind-April2017-master/pull/17#issuecomment-308226557>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AErnvrrf5_pgFOyta6ajQQ3Jn8RIV5Maks5sDuZAgaJpZM4N4pM4>.
>

-- 
Dr. Timo Baumann

Fachbereich Informatik, Universität Hamburg
AB Natürlichsprachliche Systeme

http://nats-www.informatik.uni-hamburg.de/User/TimoBaumann

timobaumann commented:
Hi Timo ,


I think I made a huge mistake during my last push when I commented out the PATH form the User Model Component.

I'm trying to fix that.

Florian


________________________________
From: Timo Baumann <baumann@informatik.uni-hamburg.de>
Sent: Tuesday, June 13, 2017 5:01 PM
To: reply@reply.github.com; Florian Pecune
Subject: Re: [fpecune/MUF-inMind-April2017-master] Sushma branch (#17)

Hi Sushma, there's a funny directory "C:\\Users\\fpecune\\Desktop\\Logs" checked in since your commit 550a83. Can you please remove that (and the files therein?)
@Florian, this file was probably created by code in ./src/main/java/edu/cmu/inmind/multiuser/sara/component/UserModelComponent.java . Can you stop that from doing this?
Thanks, t.

Am 13.06.2017 um 15:42 schrieb sushma-ananda:

Hi Timo, thanks, This is done now. updated gitignore : commit# d8a664d<https://github.com/fpecune/MUF-inMind-April2017-master/commit/d8a664d470a8633c0aa6a45ced135d10d692c865>
removed class files and few other changes: 9666980<https://github.com/fpecune/MUF-inMind-April2017-master/commit/9666980c517987512315fc643f5ff31e7f3b9208> .

-
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/fpecune/MUF-inMind-April2017-master/pull/17#issuecomment-308226557>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AErnvrrf5_pgFOyta6ajQQ3Jn8RIV5Maks5sDuZAgaJpZM4N4pM4>.


--
Dr. Timo Baumann

Fachbereich Informatik, Universität Hamburg
AB Natürlichsprachliche Systeme

http://nats-www.informatik.uni-hamburg.de/User/TimoBaumann
